### DISCUSSION TITLE
Role based authentication

### DISCUSSION THREAD

**Discussion by nirajanneupane53** [nicumicle/simple-jwt-login#18] [Q&A] [answered]
I am developing a flutter app with users having different roles . I am unable to find the roles of the users assigned. Can you guys help me out how to get the roles of the users.

**Comment by nicumicle** [comment, marked as answer]
Hello @nirajanneupane53 ,

You can check the user roles when you call the "register user" and auth/validate endpoints.


This feature is available starting with tag 3.4.2.
